### DISCUSSION TITLE
The sensitivity field ...

### DISCUSSION THREAD

**Discussion by AGHSMI** [mdolab/dafoam#97] [Tutorials] [answered]
Hello there,

Is there any convenient way to obtain the sensitivity field (D Obj / D alphaPorosity) in the Topology Optimization case, preferably for every runtime.write() instance? I am using docker at the moment ...

Many thanks in advance,

**Comment by friedenhe** [comment, marked as answer]
The sensitivity field will be printed to screen every optimization iteration. Check `Info(funcsSens)` in `calcObjFuncSens` from /home/dafoamuser/.local/lib/python3.6/site-packages/dafoam/optFuncs.py (or in the dafoam repo dafoam/dafoam/optFuncs.py). If the sensitivity has a lot of elements, it will only print the first and last five ones.

**Reply by AGHSMI** [reply]
Thank you. I found it right away.
